mail/common/src/main/java/com/fsck/k9/mail/internet/MimeHeaderEncoder.kt

```diff
@@ -17,12 +19,15 @@ object MimeHeaderEncoder {
     }
 
     private fun exceedsRecommendedLineLength(name: String, value: String): Boolean {
-        return name.length + COLON_PLUS_SPACE_LENGTH + value.length > RECOMMENDED_MAX_LINE_LENGTH
+        return name.headerPrefixLength + value.length > RECOMMENDED_MAX_LINE_LENGTH
```

Comment:
Using an extension function on `String` could lead to misunderstandings. It is not clear that it should only be used for the name and should not be applied to value.

Reply:
> It is not clear that it should only be used for the name property and should not be applied to value.

The same rationale could be applied when it's an argument to a regular function. But I changed the code to get rid of the extension function.